src/QsCompiler/CompilationManager/TypeChecking.cs

```diff
@@ -1161,10 +1167,10 @@ internal static void ImportGlobalSymbols(this FileContentManager file, Compilati
         /// Given a sequence of tree nodes, builds the corrsponding array of Q# statements (ignoring invalid fragments)
         /// using and updating the given scope context and adding the generated diagnostics to the given list of diagnostics,
         /// provided each statement consists of a suitable statement header followed by the required continuation(s), if any.
```

Comment:
Any recommendations for a concise way to include some of this text in place of "Thrown if this is not the case" in the exception tag?

Reply:
If I'm interpreting the doc comment correctly, maybe something like "Thrown if any statement is missing a header or a required continuation, or ..."?

Reply:
I'll just leave this one as is, since it's probably at least clear enough for now in the context of the summary text.

Reply:
I think it would be nice if the exception tags could be read separately without referring back to the summary too much, do you think it would be OK to actually replace the "this is not the case" part here?

Reply:
No problem! Done.